### PR TITLE
did all changes

### PR DESCRIPTION
**Pull request by MalkySinowitz**
opened new branch

### CODE REVIEW

**Review by batiFinkelshtein — APPROVED** [review]
this is excelent code!!!

File: extension-ui/manifest.json [hunk 1]
```diff
@@ -20,6 +21,9 @@
   "action": {
     "default_popup": "popup.html"
   },
+  "icons": {
+    "48": "images/icon48.png"
```
Comment (SariGlick) [inline extension-ui/manifest.json:25]: where is this file?

**Review by batiFinkelshtein — APPROVED** [review]
looks complete!

File: extension-ui/background.js [hunk 2]
```diff
@@ -82,3 +82,14 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
...
 });
+
+function showNotification(site, num) {
+  var message = NOTIFICATION_MESSAGE.replace('{site}', site).replace('{num}', num);
+  chrome.notifications.create({
+    type: 'basic',
+    iconUrl: 'images/icon48.png',
+    title: NOTIFICATION_TITLE,
+    message: message,
+    priority: 2
```
Comment (e171271) [inline extension-ui/background.js:93]: why the priority and icons are constants? maybe we will want to customize them

**Review by e171271 — CHANGES_REQUESTED** [review]

**Review by batiFinkelshtein — APPROVED** [review]
the mistakes are fixed

**Review by EA171 — APPROVED** [review]

**Review by e171271 — APPROVED** [review]

**Pull request merged**